tests/devices/test_default_qutrit.py

```diff
@@ -260,6 +260,277 @@ def test_apply_rotations_one_wire(self, qutrit_device_1_wire, tol):
+        self, qutrit_device_1_wire, tol, operation, input, expected_output, par
+    ):
+        """Tests that expectation values are properly calculated for single-wire observables with parameters."""
+
+        obs = operation(np.array(par), wires=[0])
+
+        qutrit_device_1_wire.reset()
+        qutrit_device_1_wire._state = np.array(input).reshape([3])
+        qutrit_device_1_wire.apply([], obs.diagonalizing_gates())
+        res = qutrit_device_1_wire.expval(obs)
+
+        assert np.isclose(res, expected_output, atol=tol, rtol=0)
+
+    A = np.array([[1, 0, 0], [0, -1, 0], [0, 0, 2]])
+    B = np.array([[4, 0, 0], [0, -2, 0], [0, 0, 1]])
+    obs = np.kron(A, B)
+
+    @pytest.mark.parametrize(
+        "operation,input,expected_output,par",
+        [
+            (
+                qml.THermitian,
+                [1 / math.sqrt(3), 0, 1 / math.sqrt(3), 1 / math.sqrt(3), 0, 0, 0, 0, 0],
+                1 / 3,
+                obs,
+            ),
+            (
+                qml.THermitian,
+                [0, 0, 0, 0, 0, 0, 0, 0, 1],
+                2,
+                obs,
+            ),
+            (
+                qml.THermitian,
+                [0.5, 0, 0, 0.5, 0, 0, 0, 0, 1 / math.sqrt(2)],
+                1,
+                obs,
+            ),
+            (
+                qml.THermitian,
+                [0, 0, 1 / math.sqrt(2), 1 / math.sqrt(2), 0, 0, 0, 0, 0],
+                -1.5,
+                obs,
+            ),
+            (
+                qml.THermitian,
+                np.array([1 / 3] * 9),
+                2 / 3,
+                obs,
+            ),
+            (
+                qml.THermitian,
+                [0, 1 / 2, 0, 1 / 2, 0, 1 / 2, 0, 1 / 2, 0],
+                -11 / 4,
+                obs,
+            ),
+        ],
+    )
+    def test_expval_two_wires_with_parameters(
+        self, qutrit_device_2_wires, tol, operation, input, expected_output, par
+    ):
+        """Tests that expectation values are properly calculated for two-wire observables with parameters."""
+
+        obs = operation(np.array(par), wires=[0, 1])
+
+        qutrit_device_2_wires.reset()
+        qutrit_device_2_wires._state = np.array(input).reshape([3] * 2)
+        qutrit_device_2_wires.apply([], obs.diagonalizing_gates())
+        res = qutrit_device_2_wires.expval(obs)
+        print(res, expected_output)
+        assert np.isclose(res, expected_output, atol=tol, rtol=0)
+
+    def test_expval_estimate(self):
+        """Test that the expectation value is not analytically calculated"""
+
+        dev = qml.device("default.qutrit", wires=1, shots=3)
+
+        @qml.qnode(dev)
+        def circuit():
+            return qml.expval(qml.THermitian(np.array([[0, 1, 0], [1, 0, 0], [0, 0, 1]]), wires=0))
+
+        expval = circuit()
+
+        # With 3 samples we are guaranteed to see a difference between
+        # an estimated variance an an analytically calculated one
```

Comment:
```suggestion
        # an estimated variance and an analytically calculated one
```

Reply:
The eigenvalues of this matrrix are +1, +1, -1, isn't it actually quite likely for this to give +1 as a result?